addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/cli/Parser.java

```diff
@@ -35,8 +35,7 @@
     private static final Logger logger = LoggerFactory.getLogger(Parser.class);
 
     private static final Pattern pattern = Pattern.compile("^\\s+([a-z\\s._]+)[:=]\\s*<?\"?([^>\"]+)\"?>?$");
-    private static final Pattern volumePattern = Pattern.compile(
-            "^(0|front-left):( *[0-9]+ \\/)? *([0-9]+)% *\\/? *([0-9\\-, dB]+)?(1|front-right):( *[0-9]+ \\/)? *([0-9]+)% *\\/? *([0-9\\-, dB]+)?$");
+    private static final Pattern volumePattern = Pattern.compile("^([\\w\\-]+):( *[\\d]+ \\/)? *([\\d]+)% *\\/? *([\\d\\-., dB]+)?$");
```

Comment:
As the pulseaudio strings that are parsed by this binding are rather diverse regarding spaces, newlines etc. I would prefer a minimalistic change to the regex by just adding `\.` which was the initial problem that should be solved here, right. It's just a matter of the lessons I have learned during the past years with different pulseaudio versions, that make me a little bit careful when it comes to changes to the regular expressions.